Addons/properties-templates/naviox.properties

```diff
@@ -6,6 +6,13 @@ storePasswordAsHex=true
 initialPasswordForAdmin=admin
 # There are many more password options in Configuration module of XavaPro (http://www.openxava.org/xavapro)
 
+#Amazon s3 details
+bucketName=bucketName
+accessKey=accessKey

```

Comment:
Document that both accessKey and secretkey are not required as mandatory if IAM role based access is used.